wsl-setup

```diff
@@ -112,9 +112,7 @@ win_username="${win_username// /_}"
 install_ubuntu_font
 
 # Wait for cloud-init to finish if systemd and its service is enabled.
-if status=$(LANG=C systemctl is-system-running 2>/dev/null) || [ "${status}" != "offline" ] && systemctl is-enabled --quiet cloud-init.service 2>/dev/null; then
-  cloud-init status --wait > /dev/null 2>&1 || true
-fi
+./wait-for-cloud-init
```

Comment:
Are you sure that pwd is always where the script is executed? I would rather check for current $0 directory and derive from that.

Reply:
Thanks for noticing!

I considered the possibility of `${BASH_SOURCE%/*}` instead of the more verbose alternative, but I guess it's less clear. WDYT?

Reply:
yeah, I prefer your solution too. Let’s go with it.